soco/core.py

```diff
@@ -931,10 +935,14 @@ def switch_to_line_in(self):
         Raises SoCoException (or a subclass) upon errors.
```

Comment:
The Raises stuff is also Napoleon syntax but should be written in the same way as the Returns section i.e:

```python
"""
...
Raises:
    SoCoException: (or a subclass) upon errors.
"""

and this raises section I think should be placed after the args sections